### ISSUE TITLE
Is there any way of preventing certain pixi elements' from zooming?

### ISSUE THREAD

**Issue by hevans90** [davidfig/pixi-viewport#394]
As per title, would there be any way of achieving something like this, but without changing the scale of my PIXI.Text instance here?

![malding](https://user-images.githubusercontent.com/12142999/183414928-42b0f251-12e2-4c14-bebf-adf5d4c00429.gif)

**Comment by davidfig** [comment]
Yes, listen in on the `zoomed` event off pixi-viewport, and then set scale of text to `1 / viewport.scale.x`.
